t/t0001-basic.t

```diff
@@ -409,6 +409,42 @@ test_expect_success 'no unit tests built with libtool wrapper' '
 	test_must_fail grep -q "\.libs" test_dirs
 '
 
+CMDS="\
+R \
+admin \
+cron \
+event \
+exec \
+job \
+jobs \
+jobspec \
+jobtap \
+keygen \
+kvs \
+logger \
+mini \
+module \
+ping \
+queue \
+resource \
+start \
+terminus \
+"
+
+KVS_SUBCMDS="\
+namespace \
+eventlog \
+"
+
+test_expect_success 'command --help works outside of flux instance' '
+        flux --help 2>&1 | grep -i usage &&
+        for cmd in ${CMDS}; do
+            flux ${cmd} --help 2>&1 | grep -i usage
```

Comment:
I think if one of the commands in the for loop fails, the for loop will still succeed.  See `test_valid()` in `t2200-ingest.t` for one way to do it.

Reply:
I tested by doing things like:

```
--- a/t/t0001-basic.t
+++ b/t/t0001-basic.t
@@ -429,6 +429,7 @@ queue \
 resource \
 start \
 terminus \
+foobar \
 "
```

so I thought it was safe.  But perhaps some corner cases I'm missing.  I'll to the `test_valid()` technique you did in `t2200-job-ingest`

Reply:
Right I think the last command's return code _does_ affect the result, just not the ones before that.  Your changes look good to me.